Mathlib/Analysis/NormedSpace/Basic.lean

```diff
@@ -113,63 +91,6 @@ theorem Filter.IsBoundedUnder.smul_tendsto_zero {f : ι → α} {g : ι → E} {
     (norm_smul_le y x).trans_eq (mul_comm _ _)
 #align filter.is_bounded_under.smul_tendsto_zero Filter.IsBoundedUnder.smul_tendsto_zero
 
-theorem closure_ball [NormedSpace ℝ E] (x : E) {r : ℝ} (hr : r ≠ 0) :
```

Comment:
This result and the next ones are not specifically about real vector spaces, they would work on any DenselyNormedField.

Reply:
How do I prove that there exists `c : K` such that `‖c‖ < 1` and `‖1 - c‖ < ε`?

Reply:
Argh, you're right, it's not true in general! Then I'm happy with your PR. Sorry for the noise!